qcore/timeseries.py

```diff
@@ -897,3 +897,11 @@ def acc(self, station, comp=Ellipsis):
             np.frombuffer(o, dtype=np.int32).astype(np.float32).reshape(-1, 3)[:, comp]
             / self.stations.scale[i]
         )
+
+def get_observed_stations(observed_data_folder):
```

Comment:
doc string should say what the function returns.

Reply:
@shot0829 The doc string for this still needs to be improved. A statement of what it does, and what the return value is would be good